tests/lib/prepare-restore.sh

```diff
@@ -488,7 +492,7 @@ restore_project() {
     # We use a trick to accelerate prepare/restore code in certain suites. That
```

Comment:
Should we move or reword the comment now that the implementation of the functionality is encapsulated by `delete_snapd_state`